Documentation/community/roadmap.rst

```diff
@@ -146,14 +148,16 @@ CI Test Improvements
 We have a comprehensive set of tests running in CI, but several contributors are
 currently working on `CI improvements`_ to make these more reliable and easier to
 maintain. This is a good area to get involved if you are interested in learning
-more about Cilium internals and development. 
+more about Cilium internals and development.
+
+Tetragon Security
+~~~~~~~~~~~~~~~~~
+
+Tetragon provides security observability through the JSON events and the Tetragon
```

Comment:
Security observability... and runtime enforcement, too?

Reply:
Agreed, I'll add that